File: exercism.io/python/darts/darts.py
import math

def score(x, y):
    pythagoras = math.sqrt((abs(x - 0) ** 2) + (abs(y - 0) ** 2))
    if pythagoras <= 1:
        return 10
    elif pythagoras <= 5:
        return 5
    elif pythagoras <= 10:
        return 1
    else:
        return 0

File: exercism.io/javascript/darts/darts.js
//
// This is only a SKELETON file for the 'Darts' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

export const solve = (x, y) => {
  let d = Math.sqrt((x - 0) ** 2 + (y - 0) ** 2)
  return d <= 1 ? 10 : d <= 5 ? 5 : d <= 10 ? 1 : 0
};
